components/inbound-endpoints/org.wso2.carbon.inbound.endpoint.persistence/src/main/java/org/wso2/carbon/inbound/endpoint/persistence/PersistenceUtils.java

```diff
@@ -316,22 +316,34 @@ public static int getPortOffset(Properties properties) {
         //Read the property of web socket endpoint ws.use.port.offset
         boolean usePortOffset = Boolean.valueOf(properties.getProperty(WEBSOCKET_USE_PORT_OFFSET));
         if (usePortOffset) {
-            //if its true return port offset according to the below
-            ServerConfiguration carbonConfig = ServerConfiguration.getInstance();
-            String portOffset = System.getProperty(PORT_OFFSET_SYSTEM_VAR,
-                    carbonConfig.getFirstProperty(PORT_OFFSET_CONFIG));
-            try {
-                if ((portOffset != null)) {
-                    return Integer.parseInt(portOffset.trim());
-                } else {
-                    return 0;
-                }
-            } catch (NumberFormatException e) {
-                log.error("Invalid Port Offset: " + portOffset + ". Default value 0 will be used.", e);
-                return 0;
-            }
+            //if its true return port offset accordingly
+            return getPortOffset();
         }
         // else return the 0 as it not need to have port offset
         return 0;
     }
+
+    /**
+     * Used to get the port offset value of server
```

Comment:
Doc comments need to end with a period. :)

Reply:
Fixed :)